indico/modules/events/editing/blueprint.py

```diff
@@ -53,6 +53,8 @@
                  management.RHEnabledEditableTypes, methods=('GET', 'POST'))
 _bp.add_url_rule('/editing/api/<any(paper,slides,poster):type>/editable-assignment/self-assign-enabled',
                  'api_self_assign_enabled', management.RHEditableSetSelfAssign, methods=('GET', 'PUT', 'DELETE'))
+_bp.add_url_rule('/editing/api/<any(paper,slides,poster):type>/principals', 'api_editable_type_principals',
+                 management.RHEditableTypePrincipals, methods=('GET', 'POST'))
```

Comment:
s/principals/editors/?

Reply:
Is it really wrong to use `principals` here?

Reply:
just a suggestion ;)

I tend to use "principals" when there's no more meaningful word available (let's say for a general ACL where you can assign different permissions), but go for the explicit term ("editors", etc) if one is available

Reply:
I don't want to change the naming of this everywhere across the PR, I think `principals` is fine.